src/cpu/BaseCPU.py

```diff
@@ -177,32 +177,32 @@ def takeOverFrom(self, old_cpu):
 
     tracer = Param.InstTracer(default_tracer, "Instruction tracer")
 
-    icache_port = MasterPort("Instruction Port")
-    dcache_port = MasterPort("Data Port")
+    icache_port = RequestPort("Instruction Port")
+    dcache_port = RequestPort("Data Port")
     _cached_ports = ['icache_port', 'dcache_port']
 
     if buildEnv['TARGET_ISA'] in ['x86', 'arm', 'riscv']:
         _cached_ports += ["itb.walker.port", "dtb.walker.port"]
 
-    _uncached_slave_ports = []
-    _uncached_master_ports = []
+    _uncached_response_ports = []
```

Comment:
This is *probably* ok, but let's leave it for now. I bet we can come up with a better name (maybe `uncached_interrrupt_receiving_ports`?